### DISCUSSION TITLE
Change into my own wallpaper

### DISCUSSION THREAD

**Discussion by lash0000** [prasanthrangan/hyprdots#674] [Q&A] [answered]
![image](https://github.com/prasanthrangan/hyprdots/assets/38674978/40fa96c5-f186-4809-a111-6a4dd5d36cb6)

Hello sir Konnichiwa,

I just want to thank you for your awesome dotfile that suits to my own design taste :heart: 

So I just wanna ask how can I put my own wallpapers here is it possible within the `.config` folder?

**Comment by ghost** [comment, marked as answer]
Yes you can add a custom wallpaper! The [Swww section](https://github.com/prasanthrangan/hyprdots/wiki/Theming#swww) in the hyprdots wiki explains exactly how to do that.

**Reply by lash0000** [reply]
Guess, I'll check this out thank you
